docs/machine-learning/toc.yml

```diff
@@ -1,74 +1,45 @@
-- name: ML.NET Guide
+- name: ML.NET
   href: index.yml
 - name: Overview
   expanded: true
   items:
-  - name: What is ML.NET & how does it work?
+  - name: The ML.NET API
```

Comment:
For the Overview section, would it make sense to follow a similar structure to the rest of the items in the TOC

- API
  - ML.NET 
- Tools
  - CLI
  - Model Builder